validator/project_validator.go

```diff
@@ -2280,6 +2280,15 @@ func checkTasks(project *model.Project) ValidationErrors {
 				},
 			)
 		}
+		if task.Priority >= model.MaxConfigSetPriority {
```

Comment:
I just checked and noticed this can also be set in the build variant task list, so we would also want to enforce the priority cap there as well.

Reply:
Works for me - tested both cases in staging and this also works.